### PR TITLE
GameDB: Minor info-alterations

### PR DESCRIPTION
**Pull request by pgert**

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -1203,7 +1203,7 @@ Name   = Drakan - The Ancients' Gates
 Region = PAL-M5
 Compat = 5
 EETimingHack = 1 // Fixes flickering textures.
-[patches]
+[patches = 04F9D87F]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: This will cause the patch not to work with different crc ids using the same serial.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -4395,6 +4379,14 @@ Serial = SCPS-56008
 Name   = Fatal Frame
 Region = NTSC-K
 Compat = 5
+[patches = BCF04BF3]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto.

File: bin/GameIndex.dbf [hunk 3]
```diff
@@ -8351,11 +8343,13 @@ Serial = SLES-50821
 Name   = Project Zero
 Region = PAL-M5
 Compat = 5
-[patches]
-	comment=- This gamedisc only lets you choose language once,
-	comment=-  whereafter it stores that setting in the memcard.
-	comment=- To change the language you actually have to erase
-	comment=-  the memcard content for the game.
+[patches = 22E91837]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto.

**Comment by pgert** [comment]
There isn't (to my knowledge) any other CRC for those Serials.
But if you insist ...

**Comment by pgert** [comment]
Corrected a regional-code.
https://forums.pcsx2.net/Thread-Select-language?pid=584683#pid584683

**Comment by lightningterror** [comment]
I don't think this PR is that useful.
If we look at it that way the entire GameDB needs updates like this with almost no end.

**Comment by pgert** [comment]
I find this PR to benefit End-Using, but it's your call.

**Comment by willkuer** [comment]
I'd say even a small step in the right direction is progress.

I am not sure however if referencing gsdx stuff in the gamedb is reasonable. The idea is that pcsx2 does not depend on gsdx. The usability concern however might overcome such architecture considerations. (Especially since there are no other noteworthy gs plugins)

File: bin/GameIndex.dbf [hunk 4]
```diff
@@ -1203,7 +1203,7 @@ Name   = Drakan - The Ancients' Gates
 Region = PAL-M5
 Compat = 5
 EETimingHack = 1 // Fixes flickering textures.
-[patches]
+[patches]	// 04F9D87F
 	comment=- This gamedisc supports multiple languages through the BIOS configuration.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: You can put the comment in the next row.

Example

```
// 04F9D87F
comment=- This gamedisc supports multiple languages through the BIOS configuration.
```

File: bin/GameIndex.dbf [hunk 5]
```diff
@@ -2023,20 +2011,15 @@ Region = PAL-Unk
 Serial = SCES-52677
 Name   = Network Access Disc & Hardware - Online Arena
 Region = PAL-M7
-Compat = 5
-[patches]
-	comment=- This gamedisc supports multiple languages and widescreen.
-	comment=- Language & widescreen selection is done through the BIOS configuration.
+Compat = 5	// With GSdx, SW-mode only for HOA.
+[patches = D859B1B4]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Same for this one.

File: bin/GameIndex.dbf [hunk 6]
```diff
@@ -4395,6 +4378,14 @@ Serial = SCPS-56008
 Name   = Fatal Frame
 Region = NTSC-K
 Compat = 5
+[patches]	// BCF04BF3
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto

File: bin/GameIndex.dbf [hunk 7]
```diff
@@ -8351,11 +8342,13 @@ Serial = SLES-50821
 Name   = Project Zero
 Region = PAL-M5
 Compat = 5
-[patches]
-	comment=- This gamedisc only lets you choose language once,
-	comment=-  whereafter it stores that setting in the memcard.
-	comment=- To change the language you actually have to erase
-	comment=-  the memcard content for the game.
+[patches]	// 22E91837
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto

File: bin/GameIndex.dbf [hunk 8]
```diff
@@ -22211,8 +22205,8 @@ Serial = SLPM-65232
 Name   = Devil May Cry 2 [Dante Disc]
 Region = NTSC-J
 Compat = 5
-[patches]
-	comment=Note that this disc has the same CRC as SLUS-20484, the NTSC-U disc.
+[patches = 08995DEE]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto

File: bin/GameIndex.dbf [hunk 9]
```diff
@@ -30745,6 +30739,14 @@ Serial = SLPS-25074
 Name   = Project Zero
 Region = NTSC-J
 Compat = 5
+[patches = 9883194E]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto

File: bin/GameIndex.dbf [hunk 10]
```diff
@@ -35866,6 +35868,14 @@ Serial = SLUS-20388
 Name   = Fatal Frame
 Region = NTSC-U
 Compat = 5
+[patches = 339A0B8C]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto

**Comment by lightningterror** [comment]
Can you squash the commits and improve the commit description what was changed ?
Short and simple but covers pretty much all the changes.

**Comment by pgert** [comment]
No I can't squash or alter the description.
Don't use GitHub on Windows.

File: bin/GameIndex.dbf [hunk 11]
```diff
@@ -2023,20 +2012,16 @@ Region = PAL-Unk
 Serial = SCES-52677
 Name   = Network Access Disc & Hardware - Online Arena
 Region = PAL-M7
-Compat = 5
+Compat = 5	// With GSdx, SW-mode only for HOA.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Can you remove the comment here ? I don't like it being on the compatibility line.

**Comment by refractionpcsx2** [comment]
My review still hasn't been addressed..

**Comment by refractionpcsx2** [comment]
Because I haven't completed it lol I'll complete it later when I'm on my pc

File: bin/GameIndex.dbf [hunk 12]
```diff
@@ -42927,7 +42938,7 @@ Name   = FIFA Soccer 10
 Region = NTSC-U
 ---------------------------------------------
 Serial = SLUS-21906
-Name   = Cabela's Outdoor - Adventures 2010
+Name   = Cabela's Outdoor Adventures 2009
```
[the remark this answers is not in the document]
Comment (lightningterror) [reply bin/GameIndex.dbf]: We need to update the wiki instead then.

**Comment by pgert** [comment]
Removed the compatibility comment (reluctantly).

SLUS-21192:
This speaks in favour of 2005:
https://gamefaqs.gamespot.com/ps2/929413-cabelas-outdoor-adventures-2005/data
and this - look at the disc:
https://www.ebay.com/itm/Cabelas-Outdoor-Adventures-PS2-2005-Sports-Video-Game-Scratch-Free-Disc-XD11/182627109436

SLUS-21906:
This speaks in favour of 2009:
https://gamefaqs.gamespot.com/ps2/960646-cabelas-outdoor-adventures-2009/data
and this:
https://www.play-asia.com/cabelas-outdoor-adventure-2010/13/703fzu

**Comment by refractionpcsx2** [comment]
hmm, well while neither game shows the date on them (which is annoying and confusing) I think I trust redump.org with dates more than I do gamespot (which is pretty much a wiki) as they are a preservation society, so it'd be best to match the dates on there.

**Comment by pgert** [comment]
In redump we trust.

**Comment by lightningterror** [comment]
So is this ready ?

**Comment by refractionpcsx2** [comment]
I'm happy

**Pull request merged**